tests/perf/cypress/support/e2e.ts

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/e2e.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
```

Comment:
We'll also need to support logging into a cluster with auth when running these tests. There's already a login command that you can hopefully just import here and everything will work: https://github.com/kiali/kiali/blob/master/frontend/cypress/support/commands.ts#L43-L92? :crossed_fingers:

Reply:
I will add this change.